File: ruby/arrays/sort_nested_arrays.rb
v_array = [
    1,
    [3,4], 
    [9,7,5]
]

result = Array.new 

v_array.each do |i|
    if i.is_a? Array then
        i.each do |k|
            result.insert(-1, k)
        end
    else
        result.insert(-1, i)
    end
end

result = result.sort

result.each do |j|
    puts "#{j}\n"
end

File: python/arrays/sort_nested_arrays.py
vetor = [
    [1,2],
    [8,4],
    [5,9,7]
]

result = []

for v in vetor:
    if isinstance(v, list) == True:
        for x in v:
            result.append(x)
    else:
        result.append(v)

result.sort()

for w in result:
    print(w)
